### ISSUE TITLE
Problem with Multi GPU configuration! 

### ISSUE THREAD

**Issue by Rajasekhar06** [AlexeyAB/darknet#4802]
@AlexeyAB  I have been using your work for a long time now but recently I observed that you mentioned a different calculation of hyper-parameters for Multi GPU training before that I used same parameters for single GPU and Multi GPU as I don't know that there is a change required in configuration then it worked fine but now after setting parameters according to Multi-GPU it restarts the server which has a hardware configuration of:
`TITAN RTX 24 GB (4-GPU's Turing),
INTEL XENON GOLD,
200 GB RAM
`
![Screenshot from 2019-10-30 15-12-13](https://user-images.githubusercontent.com/53042276/73814356-cfb67180-4808-11ea-94fe-443c8ee72724.png)

As far as I understood using single GPU configuration,(max-batches,steps,Learning Rate,burn-in) for Multi-GPU might divide the load among `-gpus` parameter automatically by CUDA, Can you please share your thoughts on the issue I'm facing such that it will not restart(Like I can go with Single GPU config.) **Will multi GPU config increase detection accuracy ?**.

Let's say 32 Batch size occupies 16GB GPU memory for -gpus 0
while it occupies same 16GB on each GPU if -gpus is 0,1,2,3 with single GPU config and trains model with good accuracy.
But after changing parameters as you said in [README.md](https://github.com/AlexeyAB/darknet/blob/master/README.md#how-to-train-with-multi-gpu) it stops training after certain iterations I see it restarts before **burn-in**

**Comment by Rajasekhar06** [comment]
I got rhe answer here from @AlexeyAB [here](https://github.com/WongKinYiu/CrossStagePartialNetworks/issues/11#issue-561549068) So I'm closing this issue
